### ISSUE TITLE
Resetting courses

### ISSUE THREAD

**Issue by gaborvrg** [scrimba/community#492]
It would be great if one can reset the course state. And/or easier deleting "My Notes" .

**Comment by perborgen** [comment]
Hi @gaborvrg ! 



Is this because you'd like to redo a course, and want to keep track of the progress the second time around?



Also, if I understand you correctly: you're aware that it's possible to delete Notes, but you find the current UI for doing it clunky?

**Comment by gaborvrg** [comment]
"Yes" is the answere for both questions.

**Comment by gaborvrg** [comment]
"Yes" is the answere for both questions.

**Comment by bombinaCCio** [comment]
I would like that too 👍

**Comment by minemax-ua** [comment]
And to have an "unsubscribe from a course" option would be nice as well.

**Comment by Theeemed** [comment]
Just wondered if it is possible to reset a course yet?

**Comment by piyushpranjal03** [comment]
> 

Yes now you can `unenroll` from a course home page and `re-enrol` to reset the progress.

**Comment by SZ6** [comment]
2023 and still no reset progress option?

**Comment by dezareo** [comment]
Hello, can you add "reset course" option?

**Comment by bombinaCCio** [comment]
It’s complicated, they should hire some devs 🫣

**Comment by chaselones** [comment]
I'm here also bumping this post.

**Comment by steven-oehley** [comment]
Same here, wanted to redo some react material and wanted to reset the course but no option to do so

**Comment by FranThe3rd** [comment]
reset the course lil bro

**Comment by SZ6** [comment]
starting gain to day, how do I reset the courses please?
    On Thursday 18 July 2024 at 17:29:24 BST, Francisco Figueroa ***@***.***> wrote:  
 
 


reset the course lil bro

**Comment by KDLGates** [comment]
Another vote for this feature request. Scrimba v2 has an amazing UI but to be next level I feel it needs mobile support, a few more hotkeys, and quality of life features like feeling fully in control of resetting progress meters to restart a completed, forgotten or dropped course and unsubscribing/unenrolling.
